lib/error.js

```diff
@@ -143,8 +143,12 @@ AppError.translate = function (response) {
     return response
   }
   var payload = response.output.payload
+  var reason = response.reason
   if (! payload) {
     error = new AppError({})
+  } else if (payload.statusCode === 500 && /(socket hang up|ECONNREFUSED)/.test(reason)) {
```

Comment:
also see https://github.com/mozilla/fxa-auth-server/blob/72809f8f1c93aea4ead5f25dc90977e1132374f2/lib/server.js#L33

Reply:
We did this here because this is called after logEndpointErrors is called. We didn't want to mess with the semantics/assumptions/metrics from the current logging so did this as late as possible.

Reply:
👍